e2e/pages/vscode.pages.ts

```diff
@@ -266,4 +266,43 @@ export class VSCode extends Application {
     }
     await this.window.keyboard.press('Escape');
   }
+
+  public async deleteProfile(profileNamePrefix?: string) {
+    await this.executeQuickCommand('Konveyor: Manage Analysis Profiles');
```

Comment:
```suggestion
    await this.executeQuickCommand('Konveyor: Manage Analysis Profile');
```

This changed in the latest version